spatialos-sdk/src/worker/mod.rs

```diff
@@ -45,45 +42,7 @@ impl Display for EntityId {
     }
 }
 
-#[derive(Derivative)]
-#[derivative(
-    Debug(bound = ""),
-    Copy(bound = ""),
-    Clone(bound = ""),
-    PartialEq(bound = ""),
-    Eq(bound = ""),
-    Hash(bound = "")
-)]
-pub struct RequestId<T> {
-    id: i64,
-    _type: PhantomData<*const T>,
-}
-
-impl<T> Ord for RequestId<T> {
-    fn cmp(&self, other: &Self) -> Ordering {
-        self.id.cmp(&other.id)
-    }
-}
-
-impl<T> PartialOrd for RequestId<T> {
-    fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
-        Some(self.cmp(other))
-    }
-}
-
-// SAFE: `RequestId<T>` is a type-safe wrapper around a single integer value, and is
-// therefore completely thread-safe.
-unsafe impl<T> Send for RequestId<T> {}
-unsafe impl<T> Sync for RequestId<T> {}
-
-impl<T> RequestId<T> {
-    pub fn new(id: i64) -> RequestId<T> {
-        RequestId {
-            id,
-            _type: PhantomData,
-        }
-    }
-}
+type RequestId = i64;
```

Comment:
It might still be a good idea to make `RequestId` a newtype struct that wraps an `i64`. This would at least put a bit of a type-safe boundary around request IDs, such that users are less likely to accidentally provide an invalid value where a valid request ID is required.

Reply:
Done! `RequestId` looks similar to `EntityId` now :+1: